app/views/item_seasons/show.html.erb

```diff
@@ -18,7 +18,7 @@
     <% @item_season_produces.find_each do |produce| %>
       <div class="card">
         <div class="card-inner">
-          <h4 class="produce-name"><%= link_to I18n.t("produce_items.names.#{produce.name}"), produce, class: "link_action"  %></h4>
+          <h4 class="produce-name"><%= link_to I18n.t("produce_items.names.#{produce.name}"), produce, class: "link_action produce-name"  %></h4>
```

Comment:
**question:** why does the `h4` has the same class as the link?

Reply:
I think it is a mistake, I was probably testing what worked better and left this duplication. Looking through the files, I thought it was best to remove the class from the link. Everything still works fine.